From 3178f6e41995a058eb711a7b05e92db4d9614f7d Mon Sep 17 00:00:00 2001
From: Vaibhav Hiremath <hvaibhav@ti.com>
Date: Fri, 24 Feb 2012 16:01:10 +0530
Subject: [PATCH 031/175] ARM: OMAP3: voltagedomainsam33xx_data: Remove
 unnecessary code

Based on community feedback, removed ".scalable = false"
initialization code from voltagedomain data.

Signed-off-by: Vaibhav Hiremath <hvaibhav@ti.com>
---
 arch/arm/mach-omap2/voltagedomains33xx_data.c |   17 ++++++++++-------
 1 files changed, 10 insertions(+), 7 deletions(-)

diff --git a/arch/arm/mach-omap2/voltagedomains33xx_data.c b/arch/arm/mach-omap2/voltagedomains33xx_data.c
index 7e478b6..965458d 100644
--- a/arch/arm/mach-omap2/voltagedomains33xx_data.c
+++ b/arch/arm/mach-omap2/voltagedomains33xx_data.c
@@ -1,12 +1,18 @@
 /*
  * AM33XX voltage domain data
  *
- * Copyright (C) 2011 Texas Instruments, Inc.
+ * Copyright (C) 2011 Texas Instruments Incorporated - http://www.ti.com/
  *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation version 2.
+ *
+ * This program is distributed "as is" WITHOUT ANY WARRANTY of any
+ * kind, whether express or implied; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
  */
+
 #include <linux/kernel.h>
 #include <linux/init.h>
 
@@ -14,17 +20,14 @@
 
 static struct voltagedomain am33xx_voltdm_mpu = {
 	.name		= "mpu",
-	.scalable	= false,
 };
 
 static struct voltagedomain am33xx_voltdm_core = {
 	.name		= "core",
-	.scalable	= false,
 };
 
 static struct voltagedomain am33xx_voltdm_rtc = {
 	.name		= "rtc",
-	.scalable	= false,
 };
 
 static struct voltagedomain *voltagedomains_am33xx[] __initdata = {
-- 
1.7.9.1

